src/main/java/Duke.java

```diff
@@ -1,10 +1,266 @@
+                "Here are the tasks in your list:");
+        for(Task item : list){
+            System.out.println(num+"."+item);
+            num++;
+        }
+        System.out.println("____________________________________________________________");
+    }
+
+    private static void printDone(Task task){
+        System.out.println("____________________________________________________________\n" +
+                " Nice! I've marked this task as done:\n   " +
+                task + "\n" +
+                "__________________________________________________________");
+    }
+
+    private static void printDelete(Task task, int total){
+        System.out.println("____________________________________________________________\n" +
+                " Noted. I've removed this task:\n   " +
+                task + "\n" +
+                " Now you have " + total + (total == 1 ? " task in the list.\n" : " tasks in the list.\n") +
+                "__________________________________________________________");
+    }
+
+    private static void printBye(){
+        System.out.println("____________________________________________________________\n" +
+                " Bye. Hope to see you again soon!\n" +
+                "__________________________________________________________");
+    }
+
+    private static void printError(){
+        System.out.println("____________________________________________________________\n" +
+                " ☹ OOPS!!! I'm sorry, but I don't know what that means :-(\n" +
+                "__________________________________________________________");
+    }
+
+    private static void printTodoError(){
+        System.out.println("____________________________________________________________\n" +
+                " ☹ OOPS!!! The description of a todo cannot be empty.\n" +
+                "__________________________________________________________");
+    }
+
+    private static void printDeadlineError(){
+        System.out.println("____________________________________________________________\n" +
+                " ☹ OOPS!!! The description of a deadline cannot be empty.\n" +
+                "__________________________________________________________");
+    }
+
+    private static void printEventError(){
+        System.out.println("____________________________________________________________\n" +
+                " ☹ OOPS!!! The description of a event cannot be empty.\n" +
+                "__________________________________________________________");
+    }
+
+    private static Task makeDone(List<Task> list, String input){
+        String[] message = input.split(" ");
+        int number = Integer.valueOf(message[1]);
+        list.get(number-1).markAsDone();
+        return list.get(number-1);
+    }
+
+    private static Task makeDelete(List<Task> list, String input){
+        String[] message = input.split(" ");
+        int number = Integer.valueOf(message[1]);
+        Task task = list.get(number-1);
```

Comment:
Similar problem here. The number is not checked before using it as an index of a list. It is better it can be checked.